spec/services/support/data_exports/feedback_export_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+
+RSpec.describe Support::DataExports::FeedbackExport do
+  subject { described_class.new }
+
+  context "columns" do
+    it "returns expected type" do
+      expect(subject.type).to eql("feedback")
+    end
+
+    it "returns expected headers" do
+      expect(subject.headers).to eql(["ID", "Ease of use", "User experience", "Created at"])
+    end
+  end
+
+  describe ".feedback_data" do
+    let(:feedback) { build(:feedback, id: 1, ease_of_use: "easy", experience: "Great", created_at: 1.day.ago) }
+
+    it "returns a hash of feedback attributes" do
+      res = subject.send(:feedback_data, feedback)
+      expect(res).to eql(
+        {
+          "ID" => feedback.id,
+          "Ease of use" => feedback.ease_of_use,
+          "User experience" => feedback.experience,
+          "Created at" => feedback.created_at,
+        },
+      )
+    end
+  end
+
+  describe ".data" do
```

Comment:
Instance methods we use #data :)
```suggestion
  describe "#data" do
```